### PR TITLE
adding tester + fixing mistake in 3D moves

### PR DESCRIPTION
**Pull request by pareidolies**
integrate frdescam comments + add tester + fix mistake in opengl

### CODE REVIEW

File: thistlethwaiteAlgo.py [hunk 1]
```diff
@@ -4,6 +4,17 @@
 import sys
 import time
 
+# COLORS
```
Comment (frdescam) [inline thistlethwaiteAlgo.py:7]: Maybe we should also use these colors in `RubikState`

File: main.py [hunk 2]
```diff
@@ -1,48 +1,113 @@
...
+    print("Press 's' to perform solution")
+    print("Press 'r' to reset")
 
-    input = sys.argv[1]
-    faceletsMix = getMix(input)
-    cubiesMix = input.split()
+# TESTER
 
-    rubik = RubikState()
+ALL_MOVES = ["F","B","L","R","U","D","F'","B'","L'","R'","U'","D'","F2","B2","L2","R2","U2","D2"]
```
Comment (frdescam) [inline main.py]: Wouldn't it be better to declare constants at top of the file?

File: showcase3D.py [hunk 3]
```diff
@@ -14,10 +14,10 @@
 colors = ((1, 0.8, 0), (0, 0.5, 0), (0.95, 0.95, 0.95), (0, 0, 0.75), (0.9, 0.5, 0), (0.7, 0, 0))
 moves = {
```
Comment (frdescam) [inline showcase3D.py]: Maybe put these variables in uppercase to show that they are constants

File: main.py [hunk 4]
```diff
@@ -1,48 +1,113 @@
...
 import sys
+import random
 import cProfile
 
-def main():
+# PRINT
 
-    if (len(sys.argv) != 2):
-        sys.exit('Wrong number of arguments')
+def printManual():
```
Comment (frdescam) [inline main.py:17]: Add info for ESC key

File: thistlethwaiteAlgo.py (hunk 1 above)
Comment (pareidolies) [reply thistlethwaiteAlgo.py:7]: Okay ✅

File: main.py (hunk 2 above)
Comment (pareidolies) [reply main.py]: Yes ! 👌

File: showcase3D.py (hunk 3 above)
Comment (pareidolies) [reply showcase3D.py]: Okay ✅

File: main.py (hunk 4 above)
Comment (pareidolies) [reply main.py:17]: Okay ✅

**Pull request merged**